src/scanpy/plotting/_stacked_violin.py

```diff
@@ -696,8 +696,10 @@ def stacked_violin(
     jitter: float | bool = StackedViolin.DEFAULT_JITTER,
     size: int | float = StackedViolin.DEFAULT_JITTER_SIZE,
     row_palette: str | None = StackedViolin.DEFAULT_ROW_PALETTE,
-    scale: Literal["area", "count", "width"] = StackedViolin.DEFAULT_DENSITY_NORM,
+    density_norm: Literal["area", "count", "width"] | Empty = _empty,
     yticklabels: bool = StackedViolin.DEFAULT_PLOT_YTICKLABELS,
+    # deprecated
+    scale: Literal["area", "count", "width"] | Empty = _empty,
```

Comment:
This is from an old PR? #2844? And you noticed it here? Should it be tested?

Reply:
I’ll start pulling unrelated things like this out!